modules/my_health/app/serializers/my_health/v1/prescription_details_serializer.rb

```diff
@@ -29,6 +29,7 @@ class PrescriptionDetailsSerializer < PrescriptionSerializer
       attribute :rx_rf_records
       attribute :tracking
       attribute :orderable_item
+      attribute :sorted_dispensed_date
```

Comment:
If this attribute is exposed to the client could we make sure it gets added to [the swagger schema](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/swagger/swagger/schemas/health/prescriptions.rb)? (If that's the correct schema for this endpoint) I'm not going to hold up this PR but we should add a backlog issue to get the swagger docs up to date if there isn't one already.